image_processing/Histogram Equalization/Contrast Stretch.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 28 15:22:29 2018
+
+@author: Binish125
+"""
+import cv2 
+import matplotlib.pyplot as plt
+import numpy as np
+
+last_list=[]
+rem=0
+L=256
+sk=0
+
+img=cv2.imread('inputImage.jpg',0)
```

Comment:
Change this to `img=cv2.imread('image_data/inputImage.jpg',0)`
Place images in `digital_image_processing/image_data`